bin/compile

```diff
@@ -501,9 +462,6 @@ setupProcfile() {
 
 case "${TOOL}" in
     gomodules)
-        ensureBZR "${BazaarVersion}"
```

Comment:
Should these be replaced with cache cleanup steps for now? (Otherwise existing apps will have vendored tools in their cache forever)

Reply:
@edmorley A go version change nukes the cache for this (and other reasons).

Reply:
I wondered if that might be the case before commenting above - but it only removes the `${cache}/go-path` directory, whereas these tools are saved into `${cache}/hg` and `${cache}/bzr`, so additional cache cleanup would still be required.

Reply:
See also: https://github.com/heroku/heroku-buildpack-go/blob/c644e4250314acb3686b801a18af2e2dc8cc7ce6/bin/compile#L303

Reply:
Ah true, I missed that extra line